### ISSUE TITLE
MRTK URP support for Unity 2021.3.29+ (URP 12.1.12+)

### ISSUE THREAD

**Issue by paulio** [microsoft/MixedReality-GraphicsTools-Unity#189]
## MRTK URP support for Unity 2021 (URP 12.1.12+)

Installing MRTK in a Unity 2021 project that is using URP incorrectly produces invisible materials. When the MRTK Standard opaque shader is applied the material is incorrectly invisible. The problem is not seen in Unity 2020 projects. The problem occurs for both MRTK v2 and v3. 

## To reproduce

1. Create an Unity 2021 project using the URP (3D) Template
2. Using the MRTK feature tool add the standard set of assets and the Shader examples. 
3. Use the MRTK Upgrade URP Materials tool menu (if using v2)
4. Once the install is complete view the Shader Material examples or create a new opaque material using the MRTK Standard shader
5. Note that you cannot see the resulting material in either the preview or when the material is applied to a mesh (e.g. a simple cube)

## Expected behavior

The Shader should display the color as stipulated in the shader settings

## Screenshots
Expected (from Unity 2020)
![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/235897/0dc34fa9-2d95-4f2b-afd5-86a55fdaf845)

Unity 2021
MRTK 2
![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/235897/6539f5be-7860-4666-b813-965cca0402b8)

MRTK 3
![image](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/assets/235897/a9496117-920a-4665-a391-acba537da31b)

## Your setup (please complete the following information)

- Unity Versions
2021.3.29f1
2020.3.42
- MRTK Version 
MRTK2 2.8.3
MRKT3 3.0.0-pre18

## Target platform

- HoloLens 2, UWP

**Comment by paulio** [comment]
No, the only thing I noticed was something about it not being batch compatible which was further down that inspector

**Comment by paulio** [comment]
I can confirm the repro works...
![image](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/assets/235897/36fe25ef-112a-4e0c-92b4-3f008d7d28c0)
vs. Feature Tool Install
![image](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/assets/235897/8e3e8932-484b-418b-b46c-056d19a53819)

**Comment by paulio** [comment]
Now to understand why, and see if the fix is available (or can be applied to) MRTK2 too.

**Comment by Cameron-Micka** [comment]
Interesting! @shaynie you are more knowledgeable about the feature tool than I am - do you have any ideas?

**Comment by paulio** [comment]
I compared the shaders and they seems exactly the same, apart from, maybe the dependency on the version of URP.

**Comment by paulio** [comment]
Aha, it's something to do with the URP Pipeline Asset Renderer. I copied UniversalRenderPipelineAsset & UniversalRenderPipelineAsset_Render and set them as the default in Settings URP and Settings Quality. This seems to have fixed the issue.

**Comment by paulio** [comment]
Found it...
![image](https://github.com/microsoft/MixedReality-GraphicsTools-Unity/assets/235897/a05bb823-419e-4d02-96a5-43fd5df5178b)
Change needed is from Auto to Disabled

**Comment by Cameron-Micka** [comment]
Great catch @paulio! Sounds like we need to add extra passes to the shader to support this: https://forum.unity.com/threads/custom-shaders-make-mesh-invisible-in-urp-2021-3-22f1-solved.1422327/
